assets/src/edit-story/karma/_init.js

```diff
@@ -119,6 +119,21 @@ beforeEach(() => {
   document.body.appendChild(rootEl);
   const body = rootEl.querySelector('test-body');
 
+  let clientX = -100;
+  let clientY = -100;
+  const innerCursor = document.body.appendChild(document.createElement('div'));
+  innerCursor.style =
+    'width:0px; height:0px; border-left: 10px solid red; border-bottom: 10px solid transparent; position:absolute; top:0; left:0; z-index:999; pointer-events:none;';
+  document.addEventListener('mousemove', (e) => {
```

Comment:
This should probably be `pointermove` and it should probably be with capture?

Reply:
This virtual cursor was just a test (helped me with debugging where the cursor is on small canvas), if we like it then I will apply comments and we can make it better (indicate which button was clicked/are we dragging etc.)

Reply:
I think I saw a Puppeteer plugin like this somewhere. I'll look.